File: 217-ContainsDuplicate/217-ContainsDuplicate.cpp
// Last updated: 7/2/2025, 5:43:13 PM
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {

        sort(nums.begin(),nums.end());
        if(nums.size()==1) 
        return false;
       
       
        for(int i=0;i<nums.size()-1;i++){
            if(nums[i]==nums[i+1]) 
            return true;
        }
        return false;
    }
};

File: 217-ContainsDuplicate/217-ContainsDuplicate.js
// Last updated: 7/2/2025, 5:43:07 PM
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    let seen = new Set();

    for(num of nums){
        if (seen.has(num)){
            return true
        }
        seen.add(num);
    }
     
    

 
    return false;
}
    
    
